packages/asset-service/src/service/generatedAssetData.json

```diff
@@ -75,43 +75,19 @@
     "assetId": "cosmos:osmosis-1/ibc:01E94A5FF29B8DDEFC86F412CC3927F7330E9B523CC63A6194B1108F5276025C",
     "chainId": "cosmos:osmosis-1",
     "symbol": "MEDAS",
-    "name": "MedasDigital on Osmosis",
+    "name": "Medas Digital",
     "precision": 6,
     "color": "#147CCC",
     "icon": "https://raw.githubusercontent.com/cosmos/chain-registry/master/medasdigital/images/medas.png",
     "explorer": "https://www.mintscan.io/osmosis",
     "explorerAddressLink": "https://www.mintscan.io/osmosis/account/",
     "explorerTxLink": "https://www.mintscan.io/osmosis/txs/"
   },
-  "cosmos:osmosis-1/ibc:022A879A5301CBCACF96216C967805F15C33C615B74DC7236027C1BA1BF664FF": {
-    "assetId": "cosmos:osmosis-1/ibc:022A879A5301CBCACF96216C967805F15C33C615B74DC7236027C1BA1BF664FF",
-    "chainId": "cosmos:osmosis-1",
-    "symbol": "TESTA",
-    "name": "Luca Testa FanToken on Osmosis",
-    "precision": 6,
-    "color": "#141314",
-    "icon": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/images/ft4B030260D99E3ABE2B604EA2B33BAF3C085CDA12.png",
-    "explorer": "https://www.mintscan.io/osmosis",
-    "explorerAddressLink": "https://www.mintscan.io/osmosis/account/",
-    "explorerTxLink": "https://www.mintscan.io/osmosis/txs/"
-  },
-  "cosmos:osmosis-1/ibc:051A38BBEF92B9D8293AFBE1FA293E704359E9CB28297A0FD5DBA3E9CCEE9AB1": {
-    "assetId": "cosmos:osmosis-1/ibc:051A38BBEF92B9D8293AFBE1FA293E704359E9CB28297A0FD5DBA3E9CCEE9AB1",
-    "chainId": "cosmos:osmosis-1",
-    "symbol": "ENMODA",
-    "name": "Enmoda FanToken on Osmosis",
-    "precision": 6,
-    "color": "#1A1A1A",
-    "icon": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/images/ft85AE1716C5E39EA6D64BBD7898C3899A7B500626.png",
-    "explorer": "https://www.mintscan.io/osmosis",
-    "explorerAddressLink": "https://www.mintscan.io/osmosis/account/",
-    "explorerTxLink": "https://www.mintscan.io/osmosis/txs/"
-  },
   "cosmos:osmosis-1/ibc:0954E1C28EB7AF5B72D24F3BC2B47BBB2FDF91BDDFD57B74B99E133AED40972A": {
     "assetId": "cosmos:osmosis-1/ibc:0954E1C28EB7AF5B72D24F3BC2B47BBB2FDF91BDDFD57B74B99E133AED40972A",
     "chainId": "cosmos:osmosis-1",
     "symbol": "SCRT",
-    "name": "Secret Network on Osmosis",
+    "name": "Secret Network",
```

Comment:
we've lost the `on Osmosis` bit for these tokens

Reply:
https://github.com/shapeshift/lib/pull/1157/commits/df97545b94d35e7afbe8e31af3a7252b12fbdb97